### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by xsuchy** [fedora-python/pyp2rpm#40]
This happen on F23:

```
$ pyp2rpm aj_nester --srpm -d /tmp/tmp653K_X -b 2                                                                 
INFO  Pyp2rpm initialized.                                                                                                      
INFO  Using /tmp/tmp653K_X as directory to save source.                                                                         
Traceback (most recent call last):                                                                                              
  File "/usr/bin/pyp2rpm", line 9, in <module>                                                                                  
    load_entry_point('pyp2rpm==2.0.0', 'console_scripts', 'pyp2rpm')()                                                          
  File "/usr/lib/python3.4/site-packages/click/core.py", line 716, in __call__                                                  
    return self.main(*args, **kwargs)                                                                                           
  File "/usr/lib/python3.4/site-packages/click/core.py", line 696, in main                                                      
    rv = self.invoke(ctx)                                                                                                       
  File "/usr/lib/python3.4/site-packages/click/core.py", line 889, in invoke                                                    
    return ctx.invoke(self.callback, **ctx.params)                                                                              
  File "/usr/lib/python3.4/site-packages/click/core.py", line 534, in invoke                                                    
    return callback(*args, **kwargs)                                                                                            
  File "/usr/lib/python3.4/site-packages/pyp2rpm/bin.py", line 99, in main                                                      
    converted = convertor.convert()                                                                                             
  File "/usr/lib/python3.4/site-packages/pyp2rpm/convertor.py", line 63, in convert                                             
    local_file = self.getter.get()
  File "/usr/lib/python3.4/site-packages/pyp2rpm/package_getters.py", line 85, in get
    save_file = '{0}/{1}'.format(self.save_dir, self.url.split('/')[-1])
  File "/usr/lib/python3.4/site-packages/pyp2rpm/package_getters.py", line 76, in url
    return urls[0]['url']
IndexError: list index out of range
```

**Comment by mcyprian** [comment]
Sources of this package are missing on [PyPI page](https://pypi.python.org/pypi/aj_nester/1.0.0), so it is impossible to extract metadata for the spec. However it shouldn't end up with traceback, thanks.

**Comment by mcyprian** [comment]
Already fixed 0a8abbb4cf1ba89.
